test/workflow.vader

```diff
@@ -752,44 +760,68 @@ Execute (On update):
 Execute (When already installed):
   call plug#begin()
   Plug 'junegunn/vim-easy-align', { 'do': 'touch installed2' }
-  Plug 'junegunn/vim-pseudocl', { 'do': 'touch installed2' }
```

Comment:
Oh I see. Actually didn't think about the parallel installation. Thanks for pointing it out.